public/1.0/regions/index.html

```diff
@@ -6,116 +7,117 @@
-      <li><a href="ia.json"> ia.json</a></li>
-      <li><a href="id.json"> id.json</a></li>
-      <li><a href="is.json"> is.json</a></li>
-      <li><a href="it.json"> it.json</a></li>
-      <li><a href="ja.json"> ja.json</a></li>
-      <li><a href="ka.json"> ka.json</a></li>
-      <li><a href="kab.json"> kab.json</a></li>
-      <li><a href="kk.json"> kk.json</a></li>
-      <li><a href="km.json"> km.json</a></li>
-      <li><a href="kn.json"> kn.json</a></li>
-      <li><a href="ko.json"> ko.json</a></li>
-      <li><a href="lij.json"> lij.json</a></li>
-      <li><a href="lo.json"> lo.json</a></li>
-      <li><a href="lt.json"> lt.json</a></li>
-      <li><a href="ltg.json"> ltg.json</a></li>
-      <li><a href="lv.json"> lv.json</a></li>
-      <li><a href="meh.json"> meh.json</a></li>
-      <li><a href="mk.json"> mk.json</a></li>
-      <li><a href="mr.json"> mr.json</a></li>
-      <li><a href="ms.json"> ms.json</a></li>
-      <li><a href="my.json"> my.json</a></li>
-      <li><a href="nb-NO.json"> nb-NO.json</a></li>
-      <li><a href="ne-NP.json"> ne-NP.json</a></li>
-      <li><a href="nl.json"> nl.json</a></li>
-      <li><a href="nn-NO.json"> nn-NO.json</a></li>
-      <li><a href="oc.json"> oc.json</a></li>
-      <li><a href="pa-IN.json"> pa-IN.json</a></li>
-      <li><a href="pl.json"> pl.json</a></li>
-      <li><a href="pt-BR.json"> pt-BR.json</a></li>
-      <li><a href="pt-PT.json"> pt-PT.json</a></li>
-      <li><a href="rm.json"> rm.json</a></li>
-      <li><a href="ro.json"> ro.json</a></li>
-      <li><a href="ru.json"> ru.json</a></li>
-      <li><a href="sat.json"> sat.json</a></li>
-      <li><a href="sc.json"> sc.json</a></li>
-      <li><a href="scn.json"> scn.json</a></li>
-      <li><a href="sco.json"> sco.json</a></li>
-      <li><a href="si.json"> si.json</a></li>
-      <li><a href="sk.json"> sk.json</a></li>
-      <li><a href="sl.json"> sl.json</a></li>
-      <li><a href="son.json"> son.json</a></li>
-      <li><a href="sq.json"> sq.json</a></li>
-      <li><a href="sr.json"> sr.json</a></li>
-      <li><a href="sv-SE.json"> sv-SE.json</a></li>
-      <li><a href="szl.json"> szl.json</a></li>
-      <li><a href="ta.json"> ta.json</a></li>
-      <li><a href="te.json"> te.json</a></li>
-      <li><a href="tg.json"> tg.json</a></li>
-      <li><a href="th.json"> th.json</a></li>
-      <li><a href="tl.json"> tl.json</a></li>
-      <li><a href="tr.json"> tr.json</a></li>
-      <li><a href="trs.json"> trs.json</a></li>
-      <li><a href="uk.json"> uk.json</a></li>
-      <li><a href="ur.json"> ur.json</a></li>
-      <li><a href="uz.json"> uz.json</a></li>
-      <li><a href="vi.json"> vi.json</a></li>
-      <li><a href="wo.json"> wo.json</a></li>
-      <li><a href="xh.json"> xh.json</a></li>
-      <li><a href="zh-CN.json"> zh-CN.json</a></li>
-      <li><a href="zh-TW.json"> zh-TW.json</a></li>
+      <li><a href="/1.0">Parent Directory</a></li>
+      <li><a href="en-US.json">en-US</a></li>
```

Comment:
To double check, are the reordering (putting en-US first) and removal of `.json` from the text intentional?

Reply:
I personally don't see much value in spelling out `.json` in the anchor's text, but I can restore it if needed.

As for the reordering, I actually didn't notice I was moving en-US at the top. Fixing it.

Reply:
I don't feel strongly about the `.json` so I'll merge this as-is.  Thanks!